extensions.json

```diff
@@ -1481,6 +1481,13 @@
       "download": "https://github.com/tuxtina/json2yaml/releases/download/v0.2.0/json2yaml-0.2.0.vsix",
       "version": "0.2.0"
     },
+    {
+      "id": "Tyriar.sort-lines",
+      "repository": "https://github.com/Tyriar/vscode-sort-lines",
+      "version": "1.9.0",
+      "prepublish": "npx vsce package",
+      "extensionFile": "sort-lines-1.9.0.vsix"
```

Comment:
Interesting, why are you calling `npx vsce package` here? (It is my understanding that `ovsx publish` will already do that.)

Reply:
You are right. I got it wrong. Sorry.

According to `.github/workflows/release.yml` I changed `"prepublish"` to `"yarn compile"`. `"version"` and `"extensionFile"` fields are not needed, so I removed them.

Reply:
Awesome, many thanks for double-checking! 💯